### PR TITLE
fix(support  decoded amp URLs for amp): Commit

### PR DESCRIPTION
**Pull request by Deokumarjnu**
https://github.com/quintype/ace-planning/issues/14

### CODE REVIEW

**Comment by sharangj** [comment]
Please change commit message to `fix(someStuff): Commit`

**Review by Harshith46 — APPROVED** [review]

File: src/amp-tags.js [hunk 1]
```diff
@@ -41,11 +41,12 @@ export function StoryAmpTags(seoConfig, config, pageType, data = {}, opts) {
   const { currentHostUrl = '', domainSlug } = data;
   // TODO: Remove this condition and always make absolute URL if that's better for AMP discoverability.
   const ampUrlAppend = seoConfig.appendHostToAmpUrl ? getDomain(currentHostUrl, domainSlug) || config['sketches-host'] : '';
+  const storySlug = seoConfig.decodeAmpUrl ? decodeURIComponent(story.slug): encodeURIComponent(story.slug);
```
Comment (sharangj) [inline src/amp-tags.js:44]: Can you please document this parameter?

Comment (sharangj) [reply src/amp-tags.js:44]: Otherwise it looks ok

Comment (Deokumarjnu) [reply src/amp-tags.js:44]: tried to add doc for it, but still throwing 404 in malibu doc.

**Pull request merged**